Source/Scene/Scene.js

```diff
@@ -777,16 +778,18 @@ define([
             camera.frustum.far = 10000000000.0;
         }
 
+        var pickOffscreenDefaultWidth = 0.1;
```

Comment:
Can you make this a constant or is it meant to be configurable?

Reply:
I hadn't thought about making it configurable, but I might as well. I marked it as private though since the preferred way is to pass the width to the function.